### ISSUE TITLE
Visual stepping slider changes stage size, etc.

### ISSUE THREAD

**Issue by maryfries** [jmoenig/Snap--Build-Your-Own-Blocks#1875]
I've been running into some oddities with visual stepping. Sometimes adjusting the slider makes the stage size change. Sometimes, it does the thing that's under the mouse pointer when the mouse button is released. (So, you have to keep the mouse over the slider at all times, which is unlike the behavior of most sliders.)

**Comment by maryfries** [comment]
Sometimes child blocks stay cyan even after the parent block is stopped, visual stepping is turned off, and the parent block is run completely again with visual stepping turned off. 

How do you get the cyan to go away? Is this a bug, or am I not understanding the functionality?

**Comment by brianharvey** [comment]
We know about the slider thing.  Very annoying.

The stay-cyan thing sounds like a bug.  By "child blocks" do you mean a block inside a Block Editor stepping a custom block?

**Comment by maryfries** [comment]
Yes. It's happened a few times now, stopping on different blocks depending on where it was when I stopped it. (Note that visual stepping is off in this pic.)

![image](https://user-images.githubusercontent.com/6549787/31097993-37dfb7ac-a78f-11e7-8ce5-33d6557a9783.png)
